src/components/cylc/workflow/Toolbar.vue

```diff
@@ -77,9 +80,38 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
           :disabled="!enabled.stopToggle"
           @click="onClickStop"
           size="small"
+          density="comfortable"
         />
       </div>
 
+      <!-- n-window selector -->
+      <v-btn
+        icon
+        size="small"
+        class="ml-1 mr-4"
+        ripple="false"
+      >
+        <v-chip ripple="false" size="small">
+          n={{ nWindow }}
+        </v-chip>
+        <v-menu activator="parent" :close-on-content-click="false">
+          <v-card title="Graph Window Depth">
+            <v-card-subtitle>
+              This changes the number of tasks which are displayed.
+
+              Higher values may impact performance.
+            </v-card-subtitle>
+            <v-card-text>
+              <v-select
+                density="compact"
+                v-model="nWindow"
+                :items="[0,1,2,3]"
+              />
+            </v-card-text>
+          </v-card>
+        </v-menu>
+      </v-btn>
```

Comment:
The use of a button here leads to odd styling. Can just use a chip with the `link` prop instead.
```suggestion
      <v-chip
        link
        size="small"
      >
        N = {{ nWindow }}
        <v-menu activator="parent" :close-on-content-click="false">
          <v-card title="Graph Window Depth">
            <v-card-subtitle>
              This changes the number of tasks which are displayed.

              Higher values may impact performance.
            </v-card-subtitle>
            <v-card-text>
              <v-select
                density="compact"
                v-model="nWindow"
                :items="[0,1,2,3]"
              />
            </v-card-text>
          </v-card>
        </v-menu>
      </v-chip>
```